### ISSUE TITLE
Using the browser back button throws a TypeError

### ISSUE THREAD

**Issue by christianrank** [danielr18/connected-next-router#19]
When using the browser back button, routing still works, but it throws an error:

```
Uncaught TypeError: Cannot read property 'apply' of undefined
    at Router._bps (patchRouter.js:72)
    at Router.onPopState (router.js:36)
```

It relates to the line where `_unpatchedBeforePopStateCallback` is called in `patchRouter.js`, which is not defined.

I did some research and found out that `_beforePopState` is not there anymore in Next.js:
https://github.com/zeit/next.js/commit/83334f11108fdb23246a2d5b95d20fb50697ce92#diff-a33404ba5698b3b45e69c2b3f2d3b48bL38

Then I tried replacing `_beforePopState` with `_bps` in `patchRouter.js`, but that did not help.

Maybe someone has an idea.

**Comment by danielr18** [comment]
Can you try with this line `Router.router._unpatchedBeforePopStateCallback = Router.router._bps || Router.router._beforePopState`?

**Comment by danielr18** [comment]
I just released 1.0.2, can you let me know if it's working as expected for you?

**Comment by christianrank** [comment]
Wow, that was fast.

It fixes our issue. Thanks 🎉
